examples/rdkafka_performance.c

```diff
@@ -823,12 +823,21 @@ static void do_sleep(int sleep_us) {
         }
 }
 
+static void make_random_key(char *buf, int buflen) {
+        for (int i = 0; i < buflen; i++) {
+                buf[i] = rand() >> 7; // throw away low bits which are worse
+        }
+}
+
 
 int main(int argc, char **argv) {
         char *brokers   = NULL;
         char mode       = 'C';
         char *topic     = NULL;
-        const char *key = NULL;
+        // if non-null, use this same key for every message
+        const char *fixedkey = NULL;
```

Comment:
The name change `key` -> `fixedkey` reflects the fact that this is set only if a fixed key is to be used.